### DISCUSSION TITLE
Error 2836:  The control ChangeButton on the dialog MaintenanceTypeDlg can not take focus

### DISCUSSION THREAD

**Discussion by OreCB** [wixtoolset/issues#8197] [Questions]
Hello,

I created an MSI using WiX and I'm getting this error message when I try to uninstall it by running the MSI again.
I mean, I double click the same MSI and select Remove, then I get the error message.
I also get the same error if I try to (msiexec.exe /i "mytest.msi") and select Remove, or if I try to remove it from Program and Features.

By running the (msiexec.exe /i "mytest.msi") with a log output I can see the following details for the error:

DEBUG: Error 2836:  The control ChangeButton on the dialog MaintenanceTypeDlg can not take focus
The installer has encountered an unexpected error installing this package. This may indicate a problem with this package. The error code is 2836. The arguments are: MaintenanceTypeDlg, ChangeButton, 

Now, if I run the msiexec.exe /qb /x {GUID of mytest.msi} it all works fine, it gets remove and I never see an error.

I'm using:
 Windows 10 Enterprise 
 Version 22H2
 WiX 4.0.5

I also use the "WixUI_InstallDir"
<UI>
 <ui:WixUI 
	Id="WixUI_InstallDir"
	InstallDirectory="INSTALLFOLDER"/>
</UI>

Thanks

**Comment by barnson** [comment]
Is it failing the package?
